pkg/scalers/selenium_grid_scaler.go

```diff
@@ -36,12 +36,11 @@ type seleniumGridScalerMetadata struct {
 	BrowserName         string `keda:"name=browserName,              order=triggerMetadata"`
 	SessionBrowserName  string `keda:"name=sessionBrowserName,       order=triggerMetadata, optional"`
 	ActivationThreshold int64  `keda:"name=activationThreshold,      order=triggerMetadata, optional"`
-	BrowserVersion      string `keda:"name=browserVersion,           order=triggerMetadata, optional, default=latest"`
-	UnsafeSsl           bool   `keda:"name=unsafeSsl,                order=triggerMetadata, optional, default=false"`
-	PlatformName        string `keda:"name=platformName,             order=triggerMetadata, optional, default=linux"`
-	NodeMaxSessions     int    `keda:"name=nodeMaxSessions,          order=triggerMetadata, optional, default=1"`
-
-	TargetValue int64
+	BrowserVersion      string `keda:"name=browserVersion,           order=triggerMetadata, default=latest"`
+	UnsafeSsl           bool   `keda:"name=unsafeSsl,                order=triggerMetadata, default=false"`
+	PlatformName        string `keda:"name=platformName,             order=triggerMetadata, default=linux"`
+	NodeMaxSessions     int64  `keda:"name=nodeMaxSessions,          order=triggerMetadata, default=1"`
+	TargetValue         int64  `keda:"name=targetValue,              order=triggerMetadata;resolvedEnv, default=1"`
```

Comment:
Does make sense to expose this value?  I mean, is this not almost the same as `NodeMaxSessions`?

Reply:
I missed removing it from the last trial.
Since we discussed above, scaler calculation included `nodeMaxSessions`, and `TargetValue` is constantly as 1 . No need to expose.
Revert it as kedacore/main